app/src/main/java/com/example/bigowlapp/activity/SupervisedGroupListActivity.java

```diff
@@ -96,18 +105,19 @@ public View getView(int position, View convertView, ViewGroup parent) {
         }
     }
 
-    private AlertDialog noGroupAlert() {
-        return new AlertDialog.Builder(SupervisedGroupListActivity.this)
-                .setTitle("No supervised group found!")
-                .setMessage("Required to be a supervised user of any group")
-                .setPositiveButton("Ok", (dialogInterface, which) -> SupervisedGroupListActivity.super.onBackPressed())
-                .setCancelable(false)
-                .create();
-    }
-
     @VisibleForTesting(otherwise = VisibleForTesting.NONE)
     public void setSupervisedGroupListViewModel(SupervisedGroupListViewModel supervisedGroupListViewModel) {
         this.supervisedGroupListViewModel = supervisedGroupListViewModel;
     }
+
+    @VisibleForTesting(otherwise = VisibleForTesting.NONE)
+    public AlertDialog getNoGroupAlert() {
+        return this.noGroupAlert;
+    }
+
+    @VisibleForTesting(otherwise = VisibleForTesting.NONE)
+    public Intent getIntentForTest() {
```

Comment:
If you are going to change the name of the variable, then the method's name should change.

If you are going to change the name, then you can add to the commit comment "Refactor: Change Function Declaration".

Reply:
Yup I changed the method name as well